lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -220,7 +220,7 @@ def test_create_ccx(self):
 
         # assert ccx creator has role=ccx_coach
         role = CourseCcxCoachRole(course_key)
-        self.assertTrue(role.has_user(self.coach))
+        self.assertTrue(role.has_user(self.coach, refresh=True))
```

Comment:
I'm a little confused -- what is the CCX role handling doing that makes the refresh necessary? Is it somehow side-loading itself into role info without going through the right interface? Having the `refresh` flag at all seems like a recipe for odd bugs later down the road. If CCX is mis-behaved somehow, I'd rather see it get fixed.